src/components/flex/Flex.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { CSSProperties, HTMLProps, ReactNode } from 'react';
+import cx from 'classnames';
+import theme from './theme.css';
+
+const gaps = [0, 1, 2, 3, 4, 5, 6, 7, 8] as const;
+export type Gap = typeof gaps[number];
+
+type FlexProps = Partial<
+  {
+    children: ReactNode;
+    direction: 'row' | 'column';
+    gap: Gap;
+    alignItems: 'center' | 'flex-start' | 'flex-end' | 'baseline' | 'stretch';
+    justifyContent: 'center' | 'flex-start' | 'flex-end' | 'space-around' | 'space-between' | 'space-evenly';
+    style: CSSProperties;
```

Comment:
Q: This is not defined in ticket, do we need it? If we want to add additional styling, wont it be more efficient to do it via `className` ?

Reply:
Let's discuss this in the standup.